gradle/versions.gradle

```diff
@@ -45,18 +45,21 @@ dependencyManagement {
 
     dependency 'info.picocli:picocli:4.5.0'
 
-    dependency 'io.grpc:grpc-netty:1.33.0'
+    dependency 'io.grpc:grpc-netty-shaded:1.33.1'
```

Comment:
Ugghh... shaded jars.  Is this necessary?  They have a tendency of making security based library upgrades difficult for the shaded classes.

Reply:
Asking the otel folks about this.

Reply:
Some pushback, as the grpc project relies and documents the shaded jar as their distribution of choice. That said, your concern is valid, and I'll put in the work to replace the shaded jar with the detail of dependencies.

Reply:
OK, I have taken care of the shaded jar.